src/settings/settingsmanagerhelper.h

```diff
@@ -2,6 +2,20 @@
  * @brief This header defines some macros to help SettingsManager development
  */
 
+#include <QVariant>
+
+template<bool> bool value(const QVariant& variant) {
+    return variant.toBool();
+}
+
+template<int> int value(const QVariant& variant) {
+    return variant.toInt();
+}
+
+template<typename T> T value(const QVariant& variant) {
+    return variant.value<T>();
+}
```

Comment:
Should such functions be inside a namespace to avoid conflicts with others `values` functions/templates in the future ?

Reply:
or a qVariantValueOf()